vit/help.py

```diff
@@ -110,7 +110,10 @@ def update(self, filter_args):
 
     def filter_entries(self, filter_args):
         if len(filter_args) > 0:
-            args_regex = re.compile('.*(%s).*' % '|'.join(filter_args))
+            try:
+                args_regex = re.compile('.*(%s).*' % '|'.join(filter_args))
+            except:
```

Comment:
Can we put a more specific exception in here? I prefer to catch errors more specifically so we get legitimate crashes on an unexpected error.

Reply:
yep, I left out an exception because I didn't see a widget to show an error message

Reply:
Well there are two components here:
1. Displaying an error message to the user
2. Focusing the exception on the expected error

For number one, there is a facility in ```application.py``` to display error messages, as such:
```python
        except VitException as err:
            self.activate_message_bar(str(err), 'error')
            return
```

However, the application object that exposes that method is not available in the help. I'm not sure yet of a way to work that into the help without either a fairly large refactor, or an ugly hack, I'll have to think about it.

Number two we should be able to do better with though. What is the exact error type that is thrown when ```re.compile``` fails because of a bad regex? That's what we want for the exception line, something like:
```python
except TypeError:
```
...although I'm not sure what the right error type is in this case. The reasoning there is that we want VIT to crash for any error there except the expected one we're catching.